source/includes/_memory bank.md

```diff
@@ -0,0 +1,245 @@
+      "total_ic_count" : 0,
+      "seen_ic_count" : 0,
+      "presentations_count" : 144,
+      "difficulty_bucket" : 0,
+      "scoring_goal" : 1,
+      "level" : 0.7883,
+      "score" : 0.1419,
+      "progress" : 0.1419,
+      "accuracy" : 0.5176,
+      "total_study_time_millis" : 1900256,
+      "last_study_time" : "2018-11-27T06:13:50.000Z",
+      "see_next_at" : "2018-11-26T23:00:11.000Z",
+      "due_ats" : [
+         "2018-12-01T07:59:59.000Z"
+      ],
+      "average_current_retention" : 0.0211,
+      "average_review_interval" : 60657181,
+      "average_easiness_modifier" : 0.9019,
+      "items_in_level" : {
+         "0" : 0,
+         "5" : 0,
+         "3" : 0,
+         "new" : 8,
+         "4" : 0,
+         "1" : 10,
+         "2" : 0
+      }
+    },
+    {
+      "id": 841345,
+      "name": "Ireland's Counties",
+      "module_type": "set",
+      "image": "https://assets-cerego-com.s3-us-west-2.amazonaws.com/uploads/image/uploader/1285965/harp.jpg?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIP7EH7E47533LFOA%2F20190104%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20190104T014506Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-Amz-Signature=d651b35ffb2773b02301cbea9cb1c10af3b5c33f27821bf7c099b66a80f9edaf",
+      "remixed": false,
+      "items_in_level": {
+        "0": 1,
+        "1": 20,
+        "2": 2,
+        "3": 0,
+        "4": 0,
+        "5": 0,
+        "new": 18
+      },
+      "items_count": 41,
+      "studied_items_count": 41,
+      "eligible_items_count": 23,
+      "total_ic_count": 0,
+      "seen_ic_count": 0,
+      "level": 0.8923,
+      "score": 0.8923,
+      "progress": 0.2974,
+      "scoring_goal": 3,
+      "due_ats": [],
+      "difficulty_bucket": 1,
+      "last_study_time": "2019-01-04T01:45:00.000Z",
+      "see_next_at": "2018-05-17T15:34:26.000Z",
+      "total_study_time_millis": 2676337,
+      "accuracy": 0.3738,
+      "presentations_count": 180,
+      "average_review_interval": 72220893,
+      "average_current_retention": 0.439,
+      "average_easiness_modifier": 0.8802
+    },
+    ...
+  ]
+}
+```
+
+These endpoints retrieve the goals in the memory bank for a specified user. (Can be constrained by sets within a given series)
+
+### HTTP Request
+
+`GET https://cerego.com/api/v2/users/:user_id/goals_memory_bank`
+
+`GET https://cerego.com/api/v2/users/:user_id/series/:series_id/goals_memory_bank`
+
+### Memory Object
+
+Attribute | type | Description
+--------- | ---- | -----------
+id | integer | the id of the set
+name | string | the name of the set
+module_type | string | the module type, always a "set" here
+image | string | url to the set's icon image
+remixed | boolean | whether the set is remixed or not
+items_count | integer | total number of items in the set
+studied_items_count | integer | number of items that have been studied at least once by the user
+eligible_items_count | integer | number of items that are eligible to study
```

Comment:
to make this clear -- number of items that have faded and are ready to review

Reply:
i'm....not 100% sure what eligible_items_count is -- its # of faded le_items, right?

Reply:
yeah fading memories